tests/unit/keyinput/test_basekeyparser.py

```diff
@@ -131,6 +156,15 @@ def test_keychain(self, fake_keyevent_factory):
         self.kp.handle(fake_keyevent_factory(Qt.Key_A))
         assert not self.kp.execute.called
 
+    def test_no_binding(self, monkeypatch, fake_keyevent_factory):
+        """Test special key with no binding."""
+        def none_return(binding):
+            return None
+
+        monkeypatch.setattr(utils, 'keyevent_to_string', none_return)
```

Comment:
This is a perfect place to use a [lambda](https://docs.python.org/3.4/reference/expressions.html#lambda):

`monkeypatch.setattr(utils, 'keyevent_to_string', lambda binding: None)`